frontend/packages/ceph-storage-plugin/src/components/modals/volume-snapshot-modal/volume-snapshot-modal.tsx

```diff
@@ -80,13 +81,12 @@ export const VolumeSnapshotModal = withHandlePromise((props: VolumeSnapshotModal
       fieldId="snapshot-month"
       label="Day of month"
     >
-      <TextInput
-        type="number"
-        name="snapshot-modal__month"
-        value={snapshotMonth}
-        onChange={(value) => setSnapshotMonth(Number(value))}
-        className="ceph-volume-snapshot-modal__label"
-        aria-labelledby="snapshot-month"
+      <Dropdown
+        items={days}
+        selectedKey={snapshotMonth}
+        onChange={setSnapshotMonth}
+        menuClassName="ceph-volume-snapshot-modal__dropdown--month pf-u-h-33vh"
```

Comment:
` pf-u-h-33vh` what is the use of this class ?

Reply:
to set the height of the element.

Reply:
You can set the height of the dropdown inside the custom class you defined.